src/librustc_errors/lib.rs

```diff
@@ -303,6 +303,10 @@ impl Handler {
         self.continue_after_error.set(continue_after_error);
     }
 
+    pub fn reset_err_count(&self) {
```

Comment:
Could you add a comment to this stating that it should not be called in rustc, only by tools (it would break the compiler's error handling)